DESCRIPTION

```diff
@@ -49,23 +50,22 @@ Suggests:
     kernlab,
     kknn,
     knitr,
+    mgcv,
     modeldata,
     probably,
     scales,
     spelling,
     splines2,
+    survival,
     testthat (>= 3.0.0),
     xgboost,
     xml2
 Remotes:
     tidymodels/hardhat,
     tidymodels/parsnip,
-    tidymodels/recipes,
     tidymodels/rsample,
     tidymodels/tailor,
-    tidymodels/workflows,
-    tidymodels/dials,
-    tidymodels/probably
+    tidymodels/workflows
```

Comment:
I'm loath to do this but it's probably for the best

Reply:
Hmm. That was in err. I thought I had to put a survival dependency in but don't see it now.